scripts/packages/VSCodeServer/src/display.jl

```diff
@@ -87,6 +104,8 @@ function Base.display(d::InlineDisplay, x)
                 return display(d, mime, x)
             end
         end
+    else
+        with_no_default_display(() -> display(x))
```

Comment:
```suggestion
        return with_no_default_display(() -> display(x))
```

or remove the succeeding `throw` call.